pep/app/utils/string.ts

```diff
@@ -0,0 +1,13 @@
+import { dasherize } from '@ember/string';
+
+/**
+ * Generates a cleaned up and dasherized
+ * label for accessibility use.
+ *
+ * @export
+ * @param {string} str
+ * @returns {string}
+ */
+export function createAccessibilityName(str?: string) {
```

Comment:
id maybe name this a bit more generically since it could be used for other things, maybe like `cleanAndDasherize()`?

And just note in the doc block that its removing non-word characters specifcallly.